backend/backend/settings/common.py

```diff
@@ -160,6 +160,7 @@ def build_broker_url(user: str, password: str, host: str, port: str) -> str:
 SECURE_CONTENT_TYPE_NOSNIFF = True
 SESSION_COOKIE_HTTPONLY = True
 CSRF_COOKIE_HTTPONLY = True
+CSRF_TRUSTED_ORIGINS = json.loads(os.environ.get("CSRF_TRUSTED_ORIGINS", "[]"))
```

Comment:
This setting should be set in `substra-backend/backend/backend/settings/deps/cors.py` with the other settings of the `django-cors-header` package.

Reply:
But it is related to the dependency, no ?

Reply:
Ah no, ok your link of the doc was quite confusing I thought they described config values of their module.